feat(route): add nyaa and sukebei.nyaa USER route

noname1776 opened the pull request:
new route:

- `/nyaa/user/Tsundere-Raws`
- `/nyaa/sukebei/user/Tarakara168`


## 该 PR 相关 Issue / Involved issue

no

## 完整路由地址 / Example for the proposed route(s)

<!--
请在 `routes` 区域填写以 / 开头的完整路由地址，否则你的 PR 将会被无条件关闭。
如果路由包含在文档中列出可以完全穷举的参数（例如分类），请依次全部列出。

Please include route starts with /, with all required and optional parameters. Fail to comply will result in your pull request being closed automatically.
```route
/some/route
/some/other/route
```
如果你的 PR 与路由无关, 请在 `routes` 区域 填写 `NOROUTE`，而不是直接删除 `routes` 区域。否则你的 PR 将会被无条件关闭。
If your changes are not related to route, please fill in `routes` with `NOROUTE`. Fail to comply will result in your PR being closed.
-->

```routes
/nyaa/user/Tsundere-Raws
/nyaa/sukebei/user/Tarakara168
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

noname1776 committed:
feat(route): add nyaa and sukebei.nyaa user route

new route:

- `/nyaa/user/Tsundere-Raws`
- `/nyaa/sukebei/user/Tarakara168`

TonyRL reviewed (commented):
Please also update `maintainer.js`

noname1776 committed:
fix(nyaa): add nyaa/sukebei.nyaa USER maintainer

add user route infomation in nyaa/maintainer.js
fix typo (change subekei to sukebei)
